src/eve/message/file.rs

```diff
@@ -34,6 +34,7 @@ impl std::fmt::Display for FileState {
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct FileInfo {
```

Comment:
Are there any other defaults we need?

Reply:
I don't think so. Looking at https://github.com/OISF/suricata/blob/e860b9eee96cb112a45151237d4b23fd0a9efb22/src/output-json.c#L143 it looked like it would always be present, but guess there's an optimization somewhere to just leave it as null.

Reply:
Rgr, and looks like there are a number of fields there[1][2][3] that we don't handle, assuming that's ok?

[1] https://github.com/OISF/suricata/blob/e860b9eee96cb112a45151237d4b23fd0a9efb22/src/output-json.c#L153
[2] https://github.com/OISF/suricata/blob/e860b9eee96cb112a45151237d4b23fd0a9efb22/src/output-json.c#L205
[3] https://github.com/OISF/suricata/blob/e860b9eee96cb112a45151237d4b23fd0a9efb22/src/output-json.c#L212-L213

Reply:
Yep. As it is currently, there isn't a need for them, so they have not been added.